examples/gradle.properties

```diff
@@ -1,5 +1,6 @@
 org.gradle.jvmargs=-Xmx2048M
 org.gradle.caching=true
+android.enableD8=true
```

Comment:
You should remove `examples/newsreaderExample/gradle.properties` if you enable D8 in all example apps.

Reply:
👍